augly/image/intensity.py

```diff
@@ -286,6 +285,8 @@ def sharpen_intensity(factor: float, **kwargs) -> float:
 def shuffle_pixels_intensity(factor: float, **kwargs) -> float:
     return factor * 100.0
 
+def skew_intensity(level: float, **kwargs):
+    return min(level, 2.0)
```

Comment:
the `skew_intensity` function should return a value from 0.0-100.0 -- currently, it just returns the minimum of `level` and 2.0, meaning that the output range is [-inf, 2.0], which is unideal. Can you please adjust the definition of this function?

Reply:
So can I put intensity function as previous like `return(min(level, 100))` ?

Reply:
And also this is okay for returning minimum value. 
Thanks